src/components/dialogmote/referat/Referat.tsx

```diff
@@ -205,16 +228,33 @@ const Referat = ({ dialogmote, pageTitle }: ReferatProps): ReactElement => {
             {dialogmote.behandler && <BehandlersOppgave />}
             <VeiledersOppgave />
             <StandardTekster />
+            <FlexRow topPadding={PaddingSize.SM} bottomPadding={PaddingSize.MD}>
+              <Knapp
+                htmlType="button"
+                onClick={() => setDisplayReferatPreview(true)}
+              >
+                {texts.preview}
+              </Knapp>
+            </FlexRow>
             {ferdigstillDialogmote.isError && (
               <SkjemaInnsendingFeil error={ferdigstillDialogmote.error} />
             )}
-            {submitFailed && !feilUtbedret && (
+            {mellomlagreReferat.isError && (
+              <SkjemaInnsendingFeil error={mellomlagreReferat.error} />
+            )}
+            {submitFailed && harIkkeUtbedretFeil && (
```

Comment:
Yay! Lettere å lese når ! er byttet ut med "Ikke" 🥇

Reply:
![not](https://c.tenor.com/peKSxoz9haIAAAAC/not-borat.gif)